packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx

```diff
@@ -2,24 +2,23 @@ import React, { useEffect, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { InlineLoading, InlineNotification } from '@carbon/react';
 import {
+  age,
+  attach,
   type DefaultWorkspaceProps,
   ExtensionSlot,
-  attach,
   getPatientName,
   usePatient,
-  age,
 } from '@openmrs/esm-framework';
 import styles from './ward-patient.style.scss';
+import { type WardPatientCardProps } from '../types';
 
 attach('ward-patient-workspace-header-slot', 'patient-vitals-info');
 
-export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps {
-  patientUuid: string;
-}
+interface WardPatientWorkspaceProps extends DefaultWorkspaceProps, WardPatientCardProps {}
 
-export default function WardPatientWorkspace({ patientUuid, setTitle }: WardPatientWorkspaceProps) {
+export default function WardPatientWorkspace({ setTitle, patient: { uuid } }: WardPatientWorkspaceProps) {
```

Comment:
In case someone doesn't pass the `patient` inside the workspace, `patient` will be undefined and this will surely break the code, which should be handled.
Please try not to write something, which might break.
Thanks!